sys/include/fmt.h

```diff
@@ -76,6 +91,22 @@ size_t fmt_byte_hex(char *out, uint8_t byte);
  */
 size_t fmt_bytes_hex_reverse(char *out, const uint8_t *ptr, size_t n);
 
+/**
+ * @brief Converts a sequence of hex bytes to an array of bytes
+ *
+ * The sequence of hex characters must have an even length:
+ * 2 hex character => 1 byte. If the sequence of hex has an odd length, this
+ * function returns 0 and a NULL out.
```

Comment:
`a NULL out.`:

1) use `@p out` to reference a parameter.
2) `out` is a pointer, not a pointer to a pointer, so returning `NULL` is not possible.

Reply:
done?